prow/scripts/cluster-integration/update-kyma-gke-integration.sh

```diff
@@ -0,0 +1,278 @@
+#!/usr/bin/env bash
```

Comment:
Why this script is called `update-kyma-gke-integration.sh`? It doesn't suggest that it is for testing `xip` integration.